apps/admin-gui/src/app/vos/components/applications-dynamic-list/applications-dynamic-list.component.ts

```diff
@@ -169,6 +173,41 @@ export class ApplicationsDynamicListComponent implements OnInit, OnChanges, Afte
     );
   }
 
+  exportAllData(format: string): void {
+    const sortDirection = this.sort.direction === 'asc' ? 'ASCENDING' : 'DESCENDING';
+
+    const config = getDefaultDialogConfig();
+    config.width = '300px';
+    const exportLoading = this.dialog.open(ExportDataDialogComponent, config);
+
+    this.dataSource
+      .getAllApplications(
+        this.child.paginator.length,
+        sortDirection,
+        this.getSortDataColumn(),
+        this.searchString,
+        this.includeGroupApps,
+        this.states,
+        this.dateToString(this.dateFrom),
+        this.dateToString(this.dateTo),
+        this.member?.userId ?? null,
+        this.group?.id ?? null,
+        this.getVoId(),
+        true
+      )
+      .subscribe((response) => {
+        exportLoading.close();
+        downloadData(
+          getDataForExport(
+            response,
+            this.displayedColumns,
+            this.getExportDataForColumn.bind(this) as (data: Application, column: string) => string
```

Comment:
The typing here is needed because the `getExportDataForColumn` is not static like in other components. If you would change the `getExportDataForColumn` to static, then explicit typing wouldn't be needed. 
Just an idea; feel free to counter this claim.

Reply:
Same here.